Source/WebKit/UIProcess/API/APIUIClient.h

```diff
@@ -191,7 +192,7 @@ class UIClient {
 #endif
 
 #if ENABLE(POINTER_LOCK)
-    virtual void requestPointerLock(WebKit::WebPageProxy*) { }
+    virtual void requestPointerLock(WebKit::WebPageProxy*, CompletionHandler<void(WebCore::PointerLockRequestResult)>&& completionHandler) { completionHandler(WebCore::PointerLockRequestResult::Failure); }
```

Comment:
This should just be a bool.  PointerLockRequestResult is only needed for ChomeClients.